crates/astria-sequencer/Cargo.toml

```diff
@@ -32,3 +32,7 @@ tendermint-proto = { workspace = true }
 tendermint = { workspace = true }
 tokio = { workspace = true, features = ["rt", "tracing"] }
 tracing = { workspace = true }
+
+[features]
+default = ["faucet"]
```

Comment:
Can we make this a non-default feature? I prefer that one explicitly opt into using a faucet always.